### PR TITLE
Refs [TASK][36459] list drink screen

### PR DESCRIPTION
**Pull request by VietKFC**
## Related Tickets
- [#Task 36459: List drink screen](https://edu-redmine.sun-asterisk.vn/issues/36459)
## WHAT
-
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/example/cocktailmaster/ui/category/CategoryFragment.kt [hunk 1]
```diff
@@ -5,10 +5,9 @@ import com.example.cocktailmaster.R
 import com.example.cocktailmaster.base.BaseFragment
 import com.example.cocktailmaster.data.model.Category
 import com.example.cocktailmaster.databinding.FragmentCategoryLayoutBinding
-import com.example.cocktailmaster.ui.RepositoryUtils
-import com.example.cocktailmaster.ui.hide
-import com.example.cocktailmaster.ui.popFragment
-import com.example.cocktailmaster.ui.show
+import com.example.cocktailmaster.ui.*
+import com.example.cocktailmaster.ui.listdrink.ListDrinkFragment
```
Comment (thonc-2135) [inline app/src/main/java/com/example/cocktailmaster/ui/category/CategoryFragment.kt:9]: ui.* r mà ở dưới vẫn còn import listdrink nhỉ ?

File: app/src/main/java/com/example/cocktailmaster/ui/home/HomeFragment.kt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package com.example.cocktailmaster.ui.home
 
+import android.util.Log
```
Comment (thonc-2135) [inline app/src/main/java/com/example/cocktailmaster/ui/home/HomeFragment.kt]: bỏ log

File: app/src/main/java/com/example/cocktailmaster/ui/listdrink/ListDrinkFragment.kt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    private val filterType by lazy { arguments?.getString(BUNDLE_DRINKS_FILTER) }
+    private val filterName by lazy { arguments?.getString(BUNDLE_FILTER_NAME) }
+
+    override fun initViews() {
+        binding.apply {
+            imageBack.setOnClickListener(this@ListDrinkFragment)
+            recyclerListDrink.adapter = adapter
+            ModelConstant.apply {
+                textListDrink.text = when (filterType) {
+                    FIRST_LETTER -> resources.getString(
```
Comment (thonc-2135) [inline app/src/main/java/com/example/cocktailmaster/ui/listdrink/ListDrinkFragment.kt]: ngoặc kép

File: app/src/main/java/com/example/cocktailmaster/ui/listdrink/ListDrinkFragment.kt [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    override fun showDrinks(drinks: List<Drink>) {
+        this.drinks.apply {
+            clear()
+            addAll(drinks)
+            adapter.setDrinks(this)
+        }
+    }
+
+    override fun showError() {
+
```
Comment (thonc-2135) [inline app/src/main/java/com/example/cocktailmaster/ui/listdrink/ListDrinkFragment.kt]: remove

File: app/src/main/java/com/example/cocktailmaster/ui/category/CategoryFragment.kt (hunk 1 above)
Comment (VietKFC) [reply app/src/main/java/com/example/cocktailmaster/ui/category/CategoryFragment.kt:9]: cái này e cũng ko rõ , hình như nó ko nhận file bên trong package con của ui , e import nó vẫn import thêm cái kia

**Pull request merged**